content/updates/2017-07-07-release-notes.md

```diff
@@ -0,0 +1,47 @@
++++
+date = "2017-07-07"
+title = "Platform Release Notes: July 7, 2017"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov application developer? Here are highlights from our platform updates over the past week and a half.
+<!--more-->
+
+### Added
+* Organization managers can now [add new teammates to your org]({{< relref "docs/apps/managing-teammates.md#give-roles-to-a-teammate" >}}) and manage user permissions right from the [dashboard](https://dashboard.fr.cloud.gov/).
+* [cf-service-connect]({{< relref "docs/services/relational-database.md#manually-access-a-database" >}}) now supports Windows. [See how to install it here.](https://github.com/18F/cf-service-connect#local-installation)
+
+### Changed
+
+- When you make a new service account or identity provider service instance, you will now [get the credentials using a service key]({{< relref "updates/2017-07-07-changes-to-credentials-broker.md" >}}), instead of getting the credentials from a Toaster (formerly Fugacious) link. This doesn’t change existing service account and identity provider instances, but you can delete and recreate them to use this new method.
+- [Static IP addresses for communicating with external apps]({{< relref "docs/apps/static-egress.md" >}}): Outbound traffic from cloud.gov now comes from specific IP addresses to help customers open up a connection between cloud.gov and outside data centers.
+- We recommend you upgrade your Cloud Foundry CLI to [the latest version](https://github.com/cloudfoundry/cli/releases) to get the latest bug fixes.
+- We improved the responsiveness of the dashboard to better reflect actions that happen on the backend.
+
+
+### Platform releases
+
+We upgraded the Cloud Foundry deployment to [v264](https://github.com/cloudfoundry/cf-release/releases/tag/v264). This upgrade addresses this security vulnerability: [CVE-2017-4994: Forwarded Headers in UAA](https://www.cloudfoundry.org/cve-2017-4994/).
+
+You should [restage your application](http://cli.cloudfoundry.org/en-US/cf/restage.html) to [incorporate fixes in the base filesystem](https://docs.cloudfoundry.org/devguide/deploy-apps/stacks.html#cli-commands) and ensure you’re running the most recent language version supported by your [buildpack](https://docs.cloudfoundry.org/buildpacks/).
+
+#### Additional upgrades
+- [RootFS cflinuxfs2 1.133.0](https://github.com/cloudfoundry/cflinuxfs2-rootfs-release/releases/tag/v1.133.0), which address vulnerabilities described in these security notices:
```

Comment:
This link works though: https://github.com/cloudfoundry/cflinuxfs2/releases/tag/1.133.0